### ISSUE TITLE
README.md not published in v.6.9.0

### ISSUE THREAD

**Issue by m-allanson** [stylelint/stylelint#1616]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

An issue where the readme file hasn't been published to npm for stylelint version 6.9.0. In v6.8.0 the file has been published. I had a look through [the differences](https://github.com/stylelint/stylelint/compare/6.8.0...6.9.0#files_bucket) but couldn't see any likely causes. Could this just be a temporary publish blip?

This issue prevents `npm start` from working on the [stylelint website repo](https://github.com/stylelint/stylelint.io)

> What stylelint configuration is needed to reproduce this issue?

```
npm install stylelint@6.8.0
ls node_modules/stylelint/ | grep "README"
-rw-r--r-- 6.6K 30 Jun 09:18 README.md

rm -rf ./node_modules/
npm install stylelint@6.9.0
ls node_modules/stylelint/ | grep "README"
```
> Which version of stylelint are you using?

6.9.0

> What did you expect to happen?

The file `README.md` to be included in `node_modules/stylelint` after running `npm install stylelint`

> What actually happened (e.g. what warnings or errors you are getting)?

The file was missing.

**Comment by jeddy3** [comment]
Likely caused by https://github.com/npm/npm/issues/5082

I've since updated to npm version to `3.10.x` (from `3.9.x`) and so I'm hoping this is going to away with the next patch release of stylelint.

In the meantime you can manually copy the README file into your `node_modules/stylelint` directory of your stylelint.io project. `npm start` should work then.

**Comment by m-allanson** [comment]
Fixed in 7.0.0

**Comment by ntwb** [comment]
I noticed seconds before v7 was published that over at https://www.npmjs.com/package/stylelint it only stated the latest release was v6.8.0, and not 6.9.0. I've checked and 7.0.0 is now displayed.

Not that the above was actually an issue but worth noting for reference if this issue is ever required for reference in the future...
